crypt/crypt.go

```diff
@@ -145,6 +145,9 @@ func NewFs(name, rpath string) (fs.Fs, error) {
 			return doDirChangeNotify(wrappedNotifyFunc, pollInterval)
 		}
 	}
+	if wrapSupporter, ok := wrappedFs.(fs.Wrapper); ok {
```

Comment:
This code should be in `fs/fs.go` either as a stand alone function, though it makes sense to be to put it in the `Mask` method I think.

Reply:
`I'm unsure why you'd want to do that but I'm willing to take your word for it until I see the code!`
Because `cache` is wrapped by crypt, files will always be seen in an encrypted form in the encrypted form. I need to decrypt it (the name) in cache and the only clean way to do that is ask `crypt` to decrypt it for me (inexpensive from what I can see in the code). 

I'll move that code in Mask too.